splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceDataDictionary.java

```diff
@@ -603,9 +604,10 @@ public void boot(boolean create,Properties startParams) throws StandardException
                     databaseVersion.getMinorVersionNumber(),databaseVersion.getPatchVersionNumber(),
                     databaseVersion.getSprintVersionNumber());
         }
-        if(create){
+        if(create || Boolean.TRUE.equals(EmbedConnection.isCreate.get())) {
```

Comment:
Is it possible to reuse `isCurrentlyCreatingSecondaryDatabase` here?

Reply:
Unfortunately, no, because spliceDbDesc is not yet defined at this point.

Reply:
I see. Thanks!